plugins/modules/dcnm_inventory.py

```diff
@@ -732,7 +753,6 @@ def config_save(self):
     def config_deploy(self):
 
         # config-deploy
-
```

Comment:
I see a bunch of these whitespace changes.  Please revert them if they are not needed.